src/main/java/com/hivemq/configuration/info/SystemInformation.java

```diff
@@ -75,4 +75,9 @@
      * @return the count of CPUs HiveMQ uses
      */
     int getProcessorCount();
+
+    /**
+     * @return is HiveMQ running in embedded mode
```

Comment:
Does this mean: * @return 'whether HiveMQ is running in embedded mode'?

Reply:
yes, but the *is* is a coding convention. All booleans are *is*